src/utils/langs.ts

```
@@ -14,3 +14,12 @@ export type GsLang = GsLangUser | typeof LANG_SYSTEM;
 export function getCsvDelimiter(language: string | undefined): ';' | ',' {
     return language === LANG_FRENCH ? ';' : ',';
 }
+
+export const transformIfFrenchNumber = (value: string, language: GsLang): string => {
+    value = value.trim();
```

ESLint

[error] Assignment to function parameter 'value'.

(no-param-reassign)